packages/uniforms/__suites__/index.ts

```diff
@@ -0,0 +1,5 @@
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
I'd rather move it to `/scripts/setupMatchers.ts` and add it to `setupFiles` in `jest.config.js`.